src/extensions/timeseries/TimeseriesViewNew/ElectrodeGeometryView.tsx

```diff
@@ -8,12 +8,13 @@ interface Props {
     width: number
     height: number
     selectedElectrodeIds?: number[]
+    visibleElectrodeIds: number[]
     onSelectedElectrodeIdsChanged?: (s: number[]) => void
 }
 
-const ElectrodeGeometryView: FunctionComponent<Props> = ({recordingInfo, width, height, selectedElectrodeIds, onSelectedElectrodeIdsChanged}) => {
+const ElectrodeGeometryView: FunctionComponent<Props> = ({recordingInfo, width, height, selectedElectrodeIds, visibleElectrodeIds, onSelectedElectrodeIdsChanged}) => {
     const ri = recordingInfo
-    const electrodes = useMemo(() => (ri ? zipElectrodes(ri.geom, ri.channel_ids) : []), [ri])
+    const electrodes = useMemo(() => (ri ? zipElectrodes(ri.geom, ri.channel_ids) : []).filter(a => (visibleElectrodeIds.includes(a.id))), [ri, visibleElectrodeIds])
```

Comment:
Only show the visible electrodes

Reply:
This also nicely handles zooming in the electrode geometry view, then, huh?

Reply:
yes